src/data/equiv/perm.lean

```diff
@@ -0,0 +1,96 @@
+/-
```

Comment:
Other option: make this file `group_theory/perm/basic.lean` or something like that. What do you think?

Reply:
That's not unreasonable, but I'm ambivalent. Which do you prefer?/

Reply:
Yup, that was my thinking. And `data` (originally) should be mostly about defining data, whereas this file is more about properties, lemmas, etc...
And since the `group_theory/perm` directory already exists, I think it would be natural to move this there.

Reply:
Done